astroplan/exceptions.py

```diff
@@ -0,0 +1,13 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.utils.exceptions import AstropyWarning
+
+class AlwaysUpWarning(AstropyWarning):
```

Comment:
I'd prefer something more verbose than `AlwaysUpWarning` that includes the word `Target`.
E.g. `TargetAlwaysUpWarning` or `TargetNeverSetsWarning`, because it occurs solely when the user asks when the target sets.